selftests/functional/test_loader.py

```diff
@@ -233,11 +230,11 @@ def test_runner_simple_python_like_multiple_files(self):
         cmd_line = "%s list -V %s" % (AVOCADO, mytest)
         result = process.run(cmd_line)
         self.assertIn('SIMPLE: 1', result.stdout)
-        # job should be able to finish under 5 seconds. If this fails, it's
-        # possible that we hit the "simple test fork bomb" bug
+        # Job should finish immediately unless there is a bug in Avocado
+        # which produces a fork-bomb.
```

Comment:
Most probably not, only the timeout was too tight and it's increased in this commit.